modules/ocf_mesos/manifests/package.pp

```diff
@@ -3,11 +3,20 @@
     stage => first,
   }
 
+  # at this time, v1.7.1 requires glibc >= 2.27 but stretch provides 2.24
+  $mesos_version = '1.7.0-2.0.3'
+
   # We need Java 8 to be the default java.
   include ocf::packages::java
   ocf::repackage { 'mesos':
     # mesos recommends zookeeper
-    recommends => false,
```

Comment:
I think you can still leave the trailing comma here?

Reply:
this is a holdover from something else I did, I replaced it with the semicolon